optaplanner-persistence/optaplanner-persistence-jackson/src/test/java/org/optaplanner/persistence/jackson/api/score/constraint/ConstraintMatchJacksonJsonSerializerAndDeserializerTest.java

```diff
@@ -0,0 +1,82 @@
+package org.optaplanner.persistence.jackson.api.score.constraint;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import org.junit.Before;
+import org.junit.Test;
+import org.optaplanner.core.api.score.Score;
+import org.optaplanner.core.api.score.buildin.hardsoft.HardSoftScore;
+import org.optaplanner.core.api.score.constraint.ConstraintMatch;
+import org.optaplanner.persistence.jackson.api.score.ScoreJacksonJsonSerializer;
+import org.optaplanner.persistence.jackson.api.score.buildin.hardsoft.HardSoftScoreJacksonJsonDeserializer;
+
+import static org.junit.Assert.assertEquals;
+
+public class ConstraintMatchJacksonJsonSerializerAndDeserializerTest {
+
+    ObjectMapper mapper;
```

Comment:
code encapsulation: make it private or protected. declaring nothings means "friendly", which usually not what we want